jbpm-task-assigning/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/main/java/org/kie/server/services/taskassigning/planning/SolutionBuilder.java

```diff
@@ -127,6 +131,10 @@ public TaskAssigningSolution build() {
                             addTaskToUser(assignedTasksByUserId, task, taskData.getActualOwner(), -1, pinned);
                         }
                     }
+                    break;
+                default:
+                    //no other cases exists, sonar required.
+                    break;
```

Comment:
This is where we typically throw an exception - that's defensive programming. If, in the future, you add another value and forget to change it here, that's how you'll find out.